pkg/action/dependency_test.go

```diff
@@ -159,3 +165,80 @@ func TestStatArchiveForStatus(t *testing.T) {
 	}
 	is.Equal("ok", statArchiveForStatus(where, dep))
 }
+
+func TestDependencyBuild_SkipRefresh(t *testing.T) {
+	client := NewDependency()
+	client.Verify = false
+
+	if v, ok := os.LookupEnv("GNUPGHOME"); ok {
+		client.Keyring = filepath.Join(v, "pubring.gpg")
+
+	} else {
+		client.Keyring = filepath.Join(homedir.HomeDir(), ".gnupg", "pubring.gpg")
+
+	}
```

Comment:
Why is there something about GPG in the test. The test should be about running the command while passing in the skip refresh flag. I'm not sure how that relates to GPG.